.devcontainer/postCreate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+python -m venv .venv
+source .venv/bin/activate
+python -m pip install -r requirements.txt
+python -m pip install -r requirements-dev.txt
+pre-commit install
+pre-commit run --all-files
+python -m pip install -e .
+
+# Installing during postCreateCommand instead of at docker creation
+# because foundryup installs the binaries in place not accessible by the
+# current (vscode) user.
+deactivate
```

Comment:
Is this needed? I wouldn't expect the venv to interfere with the foundry install

Reply:
iirc,  foundryup changes PATH, which is tied to the python env

Reply:
hm, looks like it works without, removed